xff/xff.go

```diff
@@ -119,18 +121,15 @@ func ParseTrustedProxies() []string {
 // New returns a set of grpc interceptors that will replace peer.IP with X-FORWARDED-FOR value if the peer ip within a subnet in allowedSubnets
 // If allowedSubnets is nil it will look for subents in the TRUSTED_PROXIES env var.
 // If allowedSubnets is nil and TRUSTED_PROXIES is empty then X-FORWARDED-FOR will be ignored (no proxy is trusted).
```

Comment:
Comment needs updating with the new name.

Reply:
🤦‍♀️ thanks for catching that